packages/mobx-react-lite/package.json

```diff
@@ -36,7 +36,9 @@
         "url": "https://github.com/mobxjs/mobx/issues"
     },
     "homepage": "https://mobx.js.org",
-    "dependencies": {},
+    "dependencies": {
+        "use-sync-external-store": "^1.2.0"
```

Comment:
Isn't this built into React 18? I'm not sure if there's a way to handle this, but if the user installs React 18, ideally we wouldn't require they also install this, no?

Reply:
We would have to maintaing seperate mobx-react versions (for >= 18 and < 18), which isn't worth it.
Note the shim prefers React's impl if available. Dunno if it's tree shakable, but it's not very big.

Reply:
I was hoping something in https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependenciesmeta would solve this problem but I'm not seeing anything unfortunately